horepg/horizon.py

```diff
@@ -95,13 +95,18 @@ def obtain(self, xmltv, channel_id, start=False, end=False):
 def parse(raw, xmltv):
     # parse raw data
     data = json.loads(raw.decode('utf-8'))
+    invalid = 0
     for listing in data['listings']:
         if not 'program' in listing:
             debug('Listing has no programme field')
             continue
         if not 'title' in listing['program']:
             debug('Listing has programme, but programme has no title')
             continue
+        if 'geen info' in listing['program']['title'].lower():
```

Comment:
This may match more then just the placeholder program information that Ziggo provides. Make it match a longer part of the title (I believe it is `Geen informatie beschikbaar`, but it could be different (I would need to check that).
Also, match using `.beginswith` to prevent matching on other program titles that may have this phrase somewhere in the name.

Or there may even be a better way (special flag) to detect this situation (but again, I would have to check the json data).

Reply:
I think in some cases the sentence is a bit different, but I cannot find an excample. I've replaced this by "geen informatie beschikbaar".

Reply:
I found an example and changed the code accordingly:

![image](https://user-images.githubusercontent.com/42780077/74592931-6b748880-5026-11ea-9109-ce67e01cfff5.png)